sstables/mx/writer.cc

```diff
@@ -594,9 +594,9 @@ class writer : public sstable_writer::writer_impl {
     run_id _run_identifier;
     bool _write_regular_as_static; // See #4139
     large_data_stats_entry _partition_size_entry;
+    large_data_stats_entry _rows_in_partition_entry;
     large_data_stats_entry _row_size_entry;
     large_data_stats_entry _cell_size_entry;
-    large_data_stats_entry _rows_in_partition_entry;
```

Comment:
Not gonna object but seems like premature optimization to me.

Reply:
we do access it frequently when writing the sstable so I think it's worth it (although I have no benchmark to prove it)

Reply:
Ok, as this doesn't make the code any harder to read/modify I won't ask for one.